app/controllers/candidate_interface/new_references/review_controller.rb

```diff
@@ -0,0 +1,58 @@
+module CandidateInterface
+  module NewReferences
+    class ReviewController < BaseController
+      def show
+        set_references
```

Comment:
Can we move this and other calls to a `before_action` ?

Reply:
👍